fairlearn/reductions/moments/conditional_opportunity.py

```diff
@@ -18,6 +27,24 @@ def init(self, dataX, dataA, dataY, dataGrp):
                            keys=["+", "-"],
                            names=["sign", "grp", "protected_attribute"])
         self.index = signed.index
+        self.default_objective_lambda_vec = None
+
+        # fill in the information about the basis
+        grp_vals = self.tags["grp"].unique()
```

Comment:
I really got confused with the naming when I started working on this, so maybe I'll open an issue to rename
grp -> group
attr -> attribute
vec -> vector (although that's the least unclear)
pos -> positive
neg -> negative

Reply:
Yes. conditional_opportunity has a variety of naming issues (beginning with the name "conditional_opportunity"). I'd like to stick to "vec" to keep things concise.